apps/client/src/shared/hooks/use-funnel.tsx

```diff
@@ -0,0 +1,63 @@
+import { useCallback } from 'react';
+import { Children } from 'react';
+import type { ReactElement, ReactNode } from 'react';
```

Comment:
```suggestion
import { useCallback, Children,  ReactElement, ReactNode } from 'react';
```
이렇게 묶어도 괜찮을것 같은데 밑에는 ReactElement와 ReactNode는 import type으로 둔 이유가 궁금해요!

Reply:
children은 실제 컴포넌트 내에서 그냥 사용하는 값이라 일반 import로 적어주었고, ReactElement, ReactNode는 타입 정의에서만 사용되는 타입이라서 import type으로 분리해서 작성했습니다.
이렇게 하면 타입 전용임이 명확히 보여서 나누어 작성했는데, 어떤 방향으로 쓰는 게 좋을까요 ..?!